client/components/fields/LinkField.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Field } from 'redux-form'
+import { connect } from 'react-redux'
+import * as selectors from '../../selectors'
+import { InputField } from './index'
+import { request } from 'axios'
+import './style.scss'
+
+export class LinkFieldComponent extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = { title: null }
+        this.errorTitle = 'Could not load title'
+        this.setTitle(this.props.link)
+    }
+
+    componentWillReceiveProps(nextProps) {
+        this.setTitle(nextProps.link)
+    }
+
+    setTitle(link) {
+        if (!link || link === 'Paste link') {
```

Comment:
Same here. Prefer to use `placeholder` attribute, or please don't hard code the message.